bin/check-nvmrc.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
```

Comment:
Note this file is written in ye olden days JavaScript so that old versions of node (even 0.10.x) can run it